omnibus/config/software/chef.rb

```diff
@@ -44,6 +44,10 @@
 dependency "chef-gem-byebug"
 dependency "chef-gem-debug_inspector"
 dependency "chef-gem-binding_of_caller"
+unless ios_xr? || solaris?
```

Comment:
Do we need both deps for any Cisco or Solaris OS? 🤔  I ask this because in the desc @rhass mentioned that the bcyrpt_pbkdf gem currently link against shared libs which are not installed on Cisco IOS-XR and that the rbnacl gems currently fail to compile on Solaris for both x86-64 and SPARC.